testutil/reindent.go

```diff
@@ -0,0 +1,63 @@
+package testutil
```

Comment:
These files should probably go to `spread/testutil/` instead of under the root directory.  We might also use `spread/internal/testutil`, but this is really a tool and we haven't made any promises that people can import the code as a library and expect it to work across versions.